Final SEO updates (#1546)
pulumi/docs: content/blog/simplify-kubernetes-rbac-in-amazon-eks-with-open-source-pulumi-packages/index.md

authors: ["nishi-davidson"]
tags: ["AWS","Kubernetes","TypeScript","EKS"]
date: "2019-04-24"

> meta_desc: "Setting up Kubernetes RBAC with Pulumi is simple, comprehensive, non-sequential and part of your everyday programming experience. This post contrasts the traditional approach with Pulumi's approach to simplifying Kubernetes RBAC in Amazon EKS."

Reviewer: ```suggestion
meta_desc: "Setting up Kubernetes RBAC with Pulumi is simple, comprehensive, and should be part of your everyday programming experience. This post contrasts the traditional approach with Pulumi's modern method for simplifying Kubernetes RBAC in Amazon EKS."
```
